src/modules/add.js

```diff
@@ -0,0 +1,19 @@
+import Display from './display.js';
+import Todo from './todos.js';
+
+export default class Add {
+  constructor(description, completed, index) {
+    this.todo = new Todo(description, completed, index);
+    this.list = document.getElementById('todo-container');
+  }
+
+  addTodo() {
+    const todos = JSON.parse(localStorage.getItem('todos')) || [];
+    this.todo.index = todos.length + 1;
```

Comment:
As the previous reviewer requested, there is a small bug you need to fix. All the other functionalities are fine but the first element is not editable. 🦾 The problem occurs because of 'todos.length +1' here. If you delete +1, it will solve the issue. Please remember the project requirement:
>By default new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).

The index value and array value doesn't match.
<img width="486" alt="Screenshot 2023-01-28 at 12 17 27 AM" src="https://user-images.githubusercontent.com/25887253/215203691-be306e98-3ebf-4639-abb5-55da448fbc6d.png">

![Screen Recording 2023-01-28 at 12 15 53 AM](https://user-images.githubusercontent.com/25887253/215203471-ca383649-622b-4fe9-b525-460829fd1b6f.gif)

Reply:
@huseyinbiyik  thanks for the review really helped...please check it now its working and if okay you can approve